cosmosdb/sql/ps-account-firewall-create.ps1

```diff
@@ -1,6 +1,6 @@
 # Create an Azure Cosmos Account for SQL (Core) API with IP Firewall
```

Comment:
This script isn't used by any file in azure-docs-pr, can it be deleted? There is already another script "common/ps-account-firewall-create.ps1" with the same functionality I guess

Reply:
I will be adding a sample page for this once it is merged.

Reply:
Yes. This one can be deleted. I will be adding a new sample page to reference this once after this is merged.